src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -280,27 +281,30 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
   @Watch('getShowErrors')
   @Watch('minAuthorizationDate') // because Authorization Date depends on this
   @Watch('authorization.date') // because Expiry Date depends on this
+  @Watch('getExistingBusinessInfo.homeIncorporationDate') // because Authorization Date validity depends on this
   private async onGetShowErrors (): Promise<void> {
     if (this.getShowErrors) {
       // wait for form to finish rendering
       await this.$nextTick()
       // validate the form and our custom components that don't support form validation
       this.$refs.formRef.validate()
-      this.$refs.authorizationDateRef.validateForm()
+      this.authorizationDateValid = this.$refs.authorizationDateRef.validateForm()
```

Comment:
I use this const to track the auth. date validity and watch it in the componentValid method